### PR TITLE
Map beta

### PR DESCRIPTION
**Pull request by atomiomi**

### CODE REVIEW

File: presets/react/rollup.config.js [hunk 1]
```diff
@@ -72,8 +72,7 @@ export default (pkg, pkgPath) => ({
       babelHelpers: 'runtime',
       rootMode: 'upward',
       exclude: [
-        path.join(pkgPath, 'node_modules/'),
-        path.join(ROOT_PKG_PATH, 'node_modules/'),
+        '**/node_modules/**/*',
```
Comment (suprMax) [inline presets/react/rollup.config.js:75]: just `exclude: /node_modules/` ?

Comment (suprMax) [reply presets/react/rollup.config.js:82]: ☝️

Comment (peterhass) [reply presets/react/rollup.config.js:82]: Not sure if I tried regex. Will check

File: presets/react/rollup.config.js [hunk 2]
```diff
@@ -55,7 +57,9 @@ export default (pkg, pkgPath) => ({
     peerDepsExternal(),
     postcss({
       extract: true,
-      modules: true,
+      modules: {
+        globalModulePaths: [/mapbox-gl/],
```
Comment (peterhass) [inline presets/react/rollup.config.js:61]: Postcss plugin doesn't have `exclude` settings like other plugins do. So I think using this should be okay.

Should we process any file from node_modules with postcss? If yes, then we have to move these settings to map's own rollup config. If not, then `/node_modules/` should be ok.

Comment (suprMax) [reply presets/react/rollup.config.js:61]: i think we have to assume same as with javascript, that node_modules includes precompiled CSS.

File: packages/map/package.json [hunk 3]
```diff
@@ -11,13 +11,13 @@
...
   },
-  "version": "0.0.0",
+  "version": "1.0.0-beta.3",
   "module": "lib/index.esm.js",
   "main": "lib/index.js",
   "sideEffects": false,
   "scripts": {
     "reinstall": "rm -rf node_modules package-lock.json && npm install",
-    "prepublishOnly": "(cd ../../ && npm run check) && npm run build",
+    "prepublishOnly": "npm run build",
```
Comment (peterhass) [inline packages/map/package.json:20]: we should keep checking before publishing

File: packages/map/src/pin_marker/index.jsx [hunk 4]
```diff
@@ -24,18 +24,19 @@ const types = [
...
 
-const images = types.reduce((acc, type) => {
-  acc[type] = require(`./images/pin-${type}.svg`);
-  return acc;
-}, {});
+// TODO: move to lib on build step
+// const images = types.reduce((acc, type) => {
+//   acc[type] = require(`./images/pin-${type}.svg`);
+//   return acc;
+// }, {});
```
Comment (peterhass) [inline packages/map/src/pin_marker/index.jsx:31]: remove?

Comment (atomiomi) [reply packages/map/src/pin_marker/index.jsx:31]: will remove in next PR

**Pull request merged**